opentech/apply/activity/messaging.py

```diff
@@ -388,6 +389,7 @@ class SlackAdapter(AdapterBase):
         MESSAGES.SEND_FOR_APPROVAL: '{user} has requested approval on project <{link}|{source.title}>.',
         MESSAGES.APPROVE_PROJECT: '{user} has approved project <{link}|{source.title}>.',
         MESSAGES.REQUEST_PROJECT_CHANGE: '{user} has requested changes for project acceptance on <{link}|{source.title}>.',
+        MESSAGES.APPROVE_CONTRACT: '{user} has approved contract for <{link}|{source.title}.',
```

Comment:
Missing activity to tell Staff that applicant has uploaded a contract for approval.